cpp/tests/table/table_tests.cu

```diff
@@ -86,106 +86,119 @@ TYPED_TEST(TableTest, ConstructColumns) {
   std::vector<gdf_dtype> dtypes{GDF_INT8, GDF_INT32, GDF_FLOAT32,
                                 cudf::gdf_dtype_of<TypeParam>()};
 
+  std::vector<gdf_dtype_extra_info> dtype_infos{
```

Comment:
This is a lot more compact. 
```suggestion
  std::vector<gdf_dtype_extra_info> dtype_infos(4, {TIME_UNIT_NONE});
```

Reply:
:+1: thanks! Suggestion included in latest commit.